master/docs/developer/database.rst

```diff
@@ -775,6 +775,7 @@ changes
     * ``changeid`` (the ID of this change)
     * ``parent_changeids`` (list of ID; change's parents)
     * ``author`` (unicode; the author of the change)
+    * ``committer`` (unicode; the committer of the change)
```

Comment:
@p12tic Already added, but I haven't mentioned that it's optional

Reply:
Hm, sorry about that then. I'm wondering how I could miss it...